ql/src/test/org/apache/hadoop/hive/ql/io/orc/TestNewInputOutputFormat.java

```diff
@@ -225,7 +225,8 @@ public void testNewOutputFormat() throws Exception {
     
     assertEquals(intWritable.get(), firstIntValue);
     assertEquals(text.toString(), firstStringValue);
```

Comment:
We could also wrap these lines in try-resource instead of closing explicitly

Reply:
This is test only code. If we would like to do this correctly, then we should use try-with-resources block for every reader. That would mean serious refactor of the affected classes. Which would mean manual backporting every change affecting those.
If you still feel that we should do it, I am happy to do the change. I wanted to keep the change small and manageable.

Thanks, Peter

Reply:
Makes sense Peter, lets keep it simple